tests/cbmc/proofs/s2n_stuffer_wipe_n/s2n_stuffer_wipe_n_harness.c

```diff
@@ -30,26 +30,40 @@ void s2n_stuffer_wipe_n_harness()
     __CPROVER_assume(s2n_result_is_ok(s2n_stuffer_validate(stuffer)));
 
     /* Save previous state. */
-    uint32_t old_n = n;
     struct s2n_stuffer old_stuffer = *stuffer;
 
+    /* Save byte from untouched portion to compare after the wipe */
+    uint32_t expect_wiped = MIN(n, old_stuffer.write_cursor);
+    uint32_t expected_write_cursor = old_stuffer.write_cursor - expect_wiped;
+    struct store_byte_from_buffer old_byte;
+    save_byte_from_array(old_stuffer.blob.data, expected_write_cursor, &old_byte);
+
     /* Function under verification. */
     if (s2n_stuffer_wipe_n(stuffer, n) == S2N_SUCCESS)
```

Comment:
Shouldn't some of these asserts also be true if s2n_stuffer_wipe_n fails? (I am aware this question is slightly out of scope for this PR.)

Reply:
Turns out, s2n_stuffer_wipe_n literally can't fail (given a valid stuffer input). I updated the proof to assert that.